packages/nx/src/command-line/migrate/migrate.ts

```diff
@@ -1216,11 +1216,12 @@ async function generateMigrationsJsonAndUpdatePackageJson(
       readNxVersion(originalPackageJson);
 
     try {
+      const [isCloudUsed, _] = isNxCloudUsed(originalNxJson);
```

Comment:
The second parameter is not used so there is no need to define it.

```suggestion
      const [isCloudUsed] = isNxCloudUsed(originalNxJson);
```